server/coordinator/coordinator_stream.go

```diff
@@ -8,10 +8,10 @@ package coordinator
 // 	"github.com/pkg/errors"
```

Comment:
delete this file?

Reply:
https://github.com/suborbital/atmo/issues/245